commands/ping.py

```diff
@@ -12,12 +12,25 @@ def __init__(self, bot):
     @commands.command(description="Displays connection speed info",
                       help='Calculates the latency of the client to the endpoint based on the time it takes for a response message to be sent to the command message. API latency information is provided by the websocket')
     async def ping(self, ctx):
-        try:
-            m = await ctx.send('Ping?')
-            await m.edit(
-                content=f'🏓 Pong! Latency is {int((snowflake_time(m.id) - snowflake_time(ctx.message.id)).microseconds / 1000)}ms. API Latency is {round(self.bot.latency * 1000)}ms')
-        except Exception as e:
-            await ctx.send(repr(e))
+        colors = {
+            299: discord.Color.green(),
+            300: 0xFFFF00,
+            500: discord.Color.orange(),
+            1000: discord.Color.red()
+        }
+        m = await ctx.send("Pinging...")
+        clr = None
+        for png, color in colors.items():
+            if png == 299 and int((snowflake_time(m.id) - snowflake_time(ctx.message.id)).microseconds / 1000) < png:
```

Comment:
`int((snowflake_time(m.id) - snowflake_time(ctx.message.id)).microseconds / 1000)`  
How about we calculate this once